libs/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -1120,3 +1120,21 @@ export const eazySmsConfig: IConfigCredentials[] = [
     description: 'Your SMS Channel Id',
   },
 ];
+
+export const whatsaAppBusinessConfig: IConfigCredentials[] = [
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please correct this typo‏
```suggestion
export const whatsAppBusinessConfig: IConfigCredentials[] = [
```

Reply:
Fixed